sdk/cosmosdb/cosmos/CHANGELOG.md

```diff
@@ -9,6 +9,8 @@
 
 ### Bugs Fixed
 
+- Fix issue with GlobalEndpointManager never makes endpoints available after they are set unavailable [#22726](https://github.com/Azure/azure-sdk-for-js/issues/22726)
```

Comment:
Terminology: We use "fall-back", please prefer to use it.

Reply:
can you elaborate a bit more?

Reply:
addressed